selene-lib/default_std/luau.yml

```diff
@@ -470,7 +476,7 @@ globals:
           display: vector
       - required: false
         type:
-            display: vector
```

Comment:
Oops, will fix!

Reply:
Actually, looks like the whitespace is bad and my formatter must have fixed it. Can take this out of this PR if you want. See https://github.com/Kampfkarren/selene/blame/d0a1622e2d2beb5a631d1f3f38b28f08c72fe33e/selene-lib/default_std/luau.yml#L473-L473C28 for when it was added.